include/onnxruntime/core/session/onnxruntime_c_api.h

```diff
@@ -836,6 +836,29 @@ struct OrtApi {
    */
   ORT_API2_STATUS(ReleaseAvailableProviders, _In_ char **ptr,
                   _In_ int providers_length);
+
+  /**
+     * \param value A tensor created from OrtCreateTensor... function.
+     * \param len element length, not including the trailing '\0' chars.
+     * \param index index of string tensor element, length of element at index will be returned.
+     */
+  ORT_API2_STATUS(GetStringTensorElementLength, _In_ const OrtValue* value, _Out_ size_t* out, size_t index);
```

Comment:
Looks like we don't always do this in every case, but the general convention for _Out_ parameters is to have them last.

Reply:
I see, the new APIs try to maintain a similar order of parameters to the existing APIs. Do you recommend we fix the order  based on convention with Out parameters at the end or should we continue to maintain somewhat the same order of parameters as the existing APIs? My rationale for retaining similar order was to have some consistency so users are not confused when using `GetStringTensorElementLength` vs `GetStringTensorDataLength` for instance.

Reply:
Yeah, if it's not too much trouble, the out parameter should be last.

Reply:
Will do